frontend/src/metadata/popups/SliceFinderPopup.svelte

```diff
@@ -39,20 +45,30 @@
 	let searchColumns = [searchColumnOptions[0]];
 
 	// Column to use as the metric to compare slices.
-	let metricColumns = $status.completeColumns.filter(
-		(d) =>
-			(d.metadataType === MetadataType.CONTINUOUS ||
-				d.metadataType === MetadataType.BOOLEAN) &&
-			notEmbedUniqCols.includes(d)
-	);
+	let metricColumns = $status.completeColumns.filter((d) => {
+		return $tab !== "comparison"
+			? (d.metadataType === MetadataType.CONTINUOUS ||
+					d.metadataType === MetadataType.BOOLEAN) &&
+					notEmbedUniqCols.includes(d)
```

Comment:
Can we pick a better name for this variable? Don't know what it is just looking at the name